xgboost/notebooks/Online Prediction with XGBoost.ipynb

```diff
@@ -0,0 +1,634 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Online Prediction with XGBoost on Cloud Machine Learning Engine\n",
+    "This notebook uses the [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to create a simple XGBoost model, upload the model to Cloud Machine Learning Engine (ML Engine), and query it for predictions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to bring your model to ML Engine\n",
+    "Getting your model ready for predictions can be done in 5 steps:\n",
+    "1. Save your model to a file\n",
+    "2. Upload the saved model to [Google Cloud Storage](https://cloud.google.com/storage)\n",
+    "3. Create a model resource on ML Engine\n",
+    "4. Create a model version (linking your XGBoost model)\n",
+    "5. Make an online prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prerequisites\n",
+    "Before we begin, let’s cover some of the different tools you’ll use to get online prediction up and running on ML Engine. \n",
+    "\n",
+    "[Google Cloud Platform](https://cloud.google.com/) (GCP) lets you build and host applications and websites, store data, and analyze data on Google's scalable infrastructure.\n",
+    "\n",
+    "[Cloud ML Engine](https://cloud.google.com/ml-engine/) is a managed service that enables you to easily build machine learning models that work on any type of data, of any size.\n",
+    "\n",
+    "[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified object storage for developers and enterprises, from live data serving to data analytics/ML to data archiving.\n",
+    "\n",
+    "[Cloud SDK](https://cloud.google.com/sdk/) is a set of tools for Cloud Platform. It contains gcloud, gsutil, and bq, which you can use to access Google Compute Engine, Google Cloud Storage, Google BigQuery, and other products and services from the command-line.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 0: Setup\n",
+    "* [Create a project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)\n",
+    "* [Create a GCS Bucket](https://cloud.google.com/storage/docs/quickstart-console)\n",
+    "* [Enable Cloud Machine Learning Engine and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component&_ga=2.217405014.1312742076.1516128282-1417583630.1516128282)\n",
+    "* [Install XGBoost](http://xgboost.readthedocs.io/en/latest/build.html)\n",
```

Comment:
These steps are good and we don't have the corresponding steps in the scikit-learn notebook?

I've created an issue to track this work: https://github.com/GoogleCloudPlatform/cloudml-samples/issues/163

Reply:
Adding the steps to Install: (scikit-learn, numpy, pandas, Google API Python Client) to scikit-learn guide